src/main/java/org/prebid/server/bidder/openx/proto/OpenxBidResponseExt.java

```diff
@@ -0,0 +1,18 @@
+package org.prebid.server.bidder.openx.proto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Value;
+
+import java.util.Map;
+
+@Value
+@AllArgsConstructor(staticName = "of")
```

Comment:
Can be replaced with `@Value(staticConstructor = "of")`

Reply:
`@Builder` appears to hide the `@Value` constructor for some reason. Could be related to https://github.com/projectlombok/lombok/issues/692 . I found `@AllArgsConstructor` to be an easy fix and not too ugly, but perhaps there's another way ?